docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -197,6 +197,51 @@ Patch releases (e.g., an update from version `1.30.1` to version `1.30.2`) ignor
 ![The scheme of using the releaseChannel parameter during Deckhouse installation and operation](images/common/deckhouse-update-process.png)
 {% endofftopic %}
 
+### Что делать, если Deckhouse не получает обновлений из канала обновлений?
```

Comment:
@kirkonru Review of the block, please

Reply:
Did some translation, please [check the commit](https://github.com/deckhouse/deckhouse/pull/5315/commits/e58ab57f1bdae62d8f576a11986b057ce2a51bee)